Luft/IDEApplicationController+Luft.m

```diff
@@ -0,0 +1,37 @@
+//
+//  IDEApplicationController+Luft.m
+//  Luft
+//
+//  Created by Emil Bogren on 15/11/15.
+//  Copyright © 2015 Hugo Tunius. All rights reserved.
+//
+
+#import "IDEApplicationController+Luft.h"
+#import "NSObject+Additions.h"
+#import "Luft.h"
+
+@implementation IDEApplicationController (Luft)
+
++ (void)luft_initialize {
+    if ([self methodForSelector:@selector(_updateEditorAndNavigateMenusIfNeeded)] == NULL) {
+        return;
+    }
+
+    [self luft_swizzleInstanceMethod:@selector(_updateEditorAndNavigateMenusIfNeeded) with:@selector(luft_updateEditorAndNavigateMenusIfNeeded)];
```

Comment:
Psst. The awesome [Aspects](https://github.com/steipete/Aspects) library is already included in the project. No need to define a swizzle method